CMakeLists.txt

```diff
@@ -244,45 +236,45 @@ if(WITH_WALL)
     endif()
 endif()
 
-# Add an option to instruct libSBML to not include SBMLTypes.h and other files internally.
-option(LIBSBML_USE_STRICT_INCLUDES
-"Compile using 'strict includes' rules: libSBML header files won't
+# Add an option to instruct libLX to not include SBMLTypes.h and other files internally.
+option(LIBLX_USE_STRICT_INCLUDES
+"Compile using 'strict includes' rules: libLX header files won't
 use SBMLTypes.h, among others." OFF)
-if(LIBSBML_USE_STRICT_INCLUDES)
-  add_definitions(-DLIBSBML_USE_STRICT_INCLUDES)
-endif(LIBSBML_USE_STRICT_INCLUDES)
-mark_as_advanced(LIBSBML_USE_STRICT_INCLUDES)
+if(LIBLX_USE_STRICT_INCLUDES)
+  add_definitions(-DLIBLX_USE_STRICT_INCLUDES)
+endif(LIBLX_USE_STRICT_INCLUDES)
+mark_as_advanced(LIBLX_USE_STRICT_INCLUDES)
 
-set(LIBSBML_BUILD_TYPE "native")
+set(LIBLX_BUILD_TYPE "native")
 if (CMAKE_SIZEOF_VOID_P EQUAL 4)
-  set(LIBSBML_BUILD_TYPE "32bit")
+  set(LIBLX_BUILD_TYPE "32bit")
 else()
-  set(LIBSBML_BUILD_TYPE "64bit")
+  set(LIBLX_BUILD_TYPE "64bit")
 endif()
 if (APPLE AND ENABLE_UNIVERSAL)
-  set(LIBSBML_BUILD_TYPE "universal")
+  set(LIBLX_BUILD_TYPE "universal")
 endif()
 
-# When allocation of memory fails or the libSBML C API is unable to open
-# a file, libSBML can exit the current process. By default this behavior
+# When allocation of memory fails or the libLX C API is unable to open
+# a file, libLX can exit the current process. By default this behavior
 # is disabled.
 #
 option (EXIT_ON_ERROR "Call exit(-1) in case of allocation or file error." OFF)
 mark_as_advanced(EXIT_ON_ERROR)
 
 # Use the version number in the shared library.
 #
-option (LIBSBML_SHARED_VERSION
+option (LIBLX_SHARED_VERSION
 "Include version information into the compiled libLX shared library." ON)
-mark_as_advanced(LIBSBML_SHARED_VERSION)
+mark_as_advanced(LIBLX_SHARED_VERSION)
 
 # Enable the generation of unit tests. If enabled, all test runners
 # will be created and can be run with "make test" or ctest.
 # This won't work in Visual Studio 2003, so we disable this option there.
 #
 if(NOT ${CMAKE_GENERATOR} MATCHES "Visual Studio 6" AND NOT ${CMAKE_GENERATOR} MATCHES "Visual Studio 7")
```

Comment:
Does it work with Visual Studio Code (assuming that is something different to Visual Studio)?

Reply:
Yes